pkg/asset/agent/installconfig_test.go

```diff
@@ -237,7 +237,7 @@ platform:
 pullSecret: "{\"auths\":{\"example.com\":{\"auth\":\"authorization value\"}}}"
 `,
 			expectedFound: false,
-			expectedError: "invalid install-config configuration: [ControlPlane.Architecture: Unsupported value: \"arm64\": supported values: \"amd64\", Compute[0].Architecture: Unsupported value: \"arm64\": supported values: \"amd64\"]",
+			expectedError: "invalid install-config configuration: [ControlPlane.Architecture: Unsupported value: \"s390x\": supported values: \"amd64\", Compute[0].Architecture: Unsupported value: \"s390x\": supported values: \"amd64\"]",
```

Comment:
Expected error will need to be updated because supported values list will  include arm64.